ext/date/date_core.c

```diff
@@ -2726,14 +2731,17 @@ date_s__valid_commercial_p(int argc, VALUE *argv, VALUE klass)
 
 /*
  * call-seq:
- *    Date.valid_commercial?(cwyear, cweek, cwday[, start=Date::ITALY])  ->  bool
+ *    Date.valid_commercial?(year, week, day, start = Date::ITALY) -> true or false
+ *
+ * Returns +true+ if the arguments define a valid commercial date,
```

Comment:
No clue.  Could be a doc bug?  (But not one that I can fix.)

Reply:
I think I will have something on this.

Reply:
For now, I've restored the original argnames, b/c they're also the names of instance methods.

Reply:
I'd like to handle this when I get to `Date.commercial`.  It may be best to discuss in a separate section, so that the 5 (or so) relevant methods can link to it.